docs/developer-docs/daos/nns/neuron-management.md

```diff
@@ -0,0 +1,272 @@
+---
+keywords: [intermediate, governance, concept, nns]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Neuron management
+
+<MarkdownChipRow labels={["Intermediate", "Governance", "Concept"]} />
+
+## Overview 
+This page describes the technical APIs used to manage neurons on the NNS.
+If you are building a project that offers neuron staking or custody support, this page is particularly useful.
+
+View the most up-to-date [`candid` definition for the NNS governance canister](https://github.com/dfinity/ic/blob/master/rs/nns/governance/canister/governance.did).
+This page will focus on the neuron interactions and explain some of the relevant definitions from this Candid file.
+To keep up to date with new NNS release proposals from DFINITY, you can subscribe to [this forum thread](https://forum.dfinity.org/t/nns-updates-aggregation-thread/23551). 
+
+## Neuron attributes
+Before explaining how to modify a neuron, it is important to understand the most important parts of a neuron.
+Every user can stake ICP utility tokens into a neuron and pariticipate in the NNS DAO with this neuron.  
+
+* **Neuron ID**: Each neuron has an identity selected by NNS governance at neuron creation.
+* **Account**: Each neuron has an associated account on the ICP ledger where the locked ICP balance resides. This account is owned by the NNS governance canister and therefore a user cannot move staked tokens.
+* **Controller**: The principal that controls the neuron. The principal must identify a public key pair, which acts as a “master key,” such that the corresponding secret key should be kept very secure. A principal can control many neurons.
```

Comment:
```suggestion
* **Controller**: The principal that controls the neuron. A principal corresponds to the public key of a public-private keypair, and anyone with possession of the corresponding private key will have full control of the neuron. Therefore, the private key of the principal ID specified as the controller should be kept very secure. A principal can control many neurons.
```
Just a thought on a different phrasing, no change required

Reply:
Adpted a slight version of this. Thanks for the suggestion